### PR TITLE
Add size to parse_vset doc

### PR DESCRIPTION
**Pull request by hesingh**
Recently, p4c added size to parser value set.  This PR changes the JSON doc to include size.

### CODE REVIEW

File: docs/JSON_format.md [hunk 1]
```diff
@@ -387,6 +387,7 @@ array item has the following attributes:
 - `id`: a unique integer (unique with respect to other parse value-sets)
 - `compressed_bitwidth`: the bitwidth of the values which can be added to the
 set. Note that this bitwidth does not include any padding.
+- `size` - number of elements in the set.
```
Comment (antoninbas) [inline docs/JSON_format.md]: comparing it with the rest of the JSON attributes (e.g. for tables) I now wish we had called it `max_size` for consistency; I believe we should still make that change

Comment (hesingh) [reply docs/JSON_format.md]: The p4c IR definition for P4ValueSet uses `size`.  So does the P4-16 spec.  However, I see that JSON format uses max_size.  I'd be happy to change p4c to use max_size. 

Regarding "I would rather not merge a PR that only documents the new attribute without implementing support for it"

Are you saying the behavioral model does not support it yet?  If that is correct, I can put this PR on hold.

Comment (hesingh) [reply docs/JSON_format.md]: See this p4c PR for maz_size JSON output.

https://github.com/p4lang/p4c/pull/2195

Comment (antoninbas) [reply docs/JSON_format.md]: If it supported it already, I would have updated this document as well. There is an open issue for this https://github.com/p4lang/behavioral-model/issues/856

**Pull request closed without merging**